### PR TITLE
Added clarifying instructions to macos infra install

### PR DESCRIPTION
**Pull request by akristen**
Per customer feedback ticket, added some content to address the lack of directory when running through these steps literally

### CODE REVIEW

File: src/content/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos.mdx [hunk 1]
```diff
@@ -35,38 +35,39 @@ To install the infrastructure monitoring agent, follow the step-by-step instruct
...
     ```
 
 5. Create the configuration file and add your <InlinePopover type="licenseKey" />:
 
     Intel x86:
 
-    ```
+    ```bash
       echo "license_key: YOUR_LICENSE_KEY" | sudo tee -a /usr/local/etc/newrelic-infra/newrelic-infra.yml
     ```
```
Comment (josemore) [inline src/content/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos.mdx:66]:
```suggestion
    ```bash
      sudo mkdir -p /usr/local/etc/newrelic-infra/
      echo "license_key: YOUR_LICENSE_KEY" | sudo tee -a /usr/local/etc/newrelic-infra/newrelic-infra.yml
    ```
```

File: src/content/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos.mdx [hunk 2]
```diff
@@ -35,38 +35,39 @@ To install the infrastructure monitoring agent, follow the step-by-step instruct
...
 
     Apple Silicon:
-    ```
+    
+    ```bash
       echo "license_key: YOUR_LICENSE_KEY" | sudo tee -a /opt/homebrew/etc/newrelic-infra/newrelic-infra.yml
```
Comment (josemore) [inline src/content/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos.mdx:71]:
```suggestion
    ```bash
      sudo mkdir -p /opt/homebrew/etc/newrelic-infra/
      echo "license_key: YOUR_LICENSE_KEY" | sudo tee -a /opt/homebrew/etc/newrelic-infra/newrelic-infra.yml
```

File: src/content/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos.mdx [hunk 3]
```diff
@@ -35,38 +35,39 @@ To install the infrastructure monitoring agent, follow the step-by-step instruct
...
 
     If it's not installed, you can install it with this command (or check [Homebrew](https://brew.sh/) up-to-date instructions):
 
-    ```
+    ```bash
       /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
     ```
 
-3. Then, open the terminal and run the following command:
+3. In your terminal, run `mkdir -p` to create a new directory for the infrastructure agent, then run the following command:
```
Comment (josemore) [inline src/content/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos.mdx:48]: Added the mkdir command together with the file creation below, we can dismiss this change.

**Review by jeff-colucci — APPROVED** [review]
Looks good!

**Pull request merged**